plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -209,14 +201,18 @@ func (adx *AzureDataExplorer) createAzureDataExplorerTable(ctx context.Context,
 		adx.Log.Info("skipped table creation")
 		return nil
 	}
-	createStmt := kusto.NewStmt("", kusto.UnsafeStmt(unsafe.Stmt{Add: true, SuppressWarning: true})).UnsafeAdd(fmt.Sprintf(createTableCommand, tableName))
-	if _, err := adx.kustoClient.Mgmt(ctx, adx.Database, createStmt); err != nil {
+
+	createStmt := kql.New(createTableCommand)
+	createParams := kusto.QueryParameters(kql.NewParameters().AddString("TABLE", tableName))
```

Comment:
You dropped the unsafe because we now build the string as a parameter?

Reply:
Exactly. Unsafe is also deprecated and the linter goes havoc. So I thought if we turn-over the code anyway, let's also get rid of this.